sympy/combinatorics/pc_groups.py

```diff
@@ -0,0 +1,67 @@
+from sympy.combinatorics.fp_groups import FpGroup, FpSubgroup, subgroup_quotient
+from sympy import sieve
+
+class PcGroup(FpGroup):
+    '''
+    Class implementing Polycyclic groups.
+    To-do : Docstring completion.
+            Add error and exception handling
+    '''
+    def __init__(fp_group, pc_series=None, pcgs=None):
+        self.fp_group = fp_group
+        self.pc_series = pc_series
+        self.pcgs = pcgs
```

Comment:
That's not a very enlightening name. `pc_sequence` or `pc_gens` would be better.